modules/virt-aws-bm.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-aws-bm_{context}"]
+= {VirtProductName} on AWS bare metal
+
+You can run {VirtProductName} on an Amazon Web Services (AWS) bare metal {product-title} cluster. Before you set up your cluster, review the following summary of supported features and limitations:
+
+Installing::
+
+* You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare metal node instance types in the `install-config.yaml` file. For example, you can use the `c6g.metal` type value for a machine based on 64-bit ARM architecture.
+** For more information, see the {product-title} documentation about installing on AWS.
+
+Accessing virtual machines (VMs)::
+
+* There is no change to how you access VMs by using the `virtctl` CLI tool or the {product-title} web console.
+* You can expose VMs by using a `NodePort` or `LoadBalancer` service. 
+** The load balancer approach is preferable because {product-title} automatically creates the load balancer in AWS and manages its lifecycle. A security group is also created for the load balancer, and you can use annotations to attach existing security groups. When you remove the service, {product-title} removes the load balancer and its associated resources.
+
+Networking::
+
+* Because connecting a secondary network to a VM is not supported on AWS, you must use the primary network, which you can expose by using services. 
+* If your application requires a flat layer 2 network or control over the IP pool, consider using OVN-Kubernetes secondary overlay networks.
```

Comment:
@AlonaKaplan @orenc1  Do we know that this works well on AWS? In particular, does AWS clusters have MTU 9000?

Also, please add a direct link to using ovn overlays, it's not easy to find.

Reply:
Yes, in AWS clusters the MTU is 9000 bytes. Because of ovn-k overhead, the workloads see a net MTU size of 8901:
```
apiVersion: operator.openshift.io/v1
kind: Network
metadata:
  annotations:
    networkoperator.openshift.io/ovn-cluster-initiator: 10.0.20.132
  name: cluster
spec:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  defaultNetwork:
    ovnKubernetesConfig:
      egressIPConfig: {}
      gatewayConfig:
        routingViaHost: false
      genevePort: 6081
      mtu: 8901                  # <--
      policyAuditConfig:
        destination: "null"
        maxFileSize: 50
        maxLogFiles: 5
        rateLimit: 20
        syslogFacility: local0
    type: OVNKubernetes
  deployKubeProxy: false
  disableMultiNetwork: false
  disableNetworkDiagnostics: false
  logLevel: Normal
  managementState: Managed
  observedConfig: null
  operatorLogLevel: Normal
  serviceNetwork:
  - 172.30.0.0/16
  unsupportedConfigOverrides: null
  useMultiNetworkPolicy: false
```

Reply:
Thanks, @orenc1 .
@ousleyp can you add a link to "using OVN-Kubernetes secondary overlay networks" ?

Reply:
@dankenigsberg There is actually already a link (see underneath the "Unsupported storage" section - there is a small "Additional resources" section). [hoping that this link to that section works](https://66765--docspreview.netlify.app/openshift-enterprise/latest/virt/install/preparing-cluster-for-virt#virt-hardware-os-requirements_preparing-cluster-for-virt:~:text=support%20live%20migration.-,Additional%20resources,-Connecting%20a%20virtual)

I totally agree that it is not ideal. Due to our publishing constraints, we can't put cross-references inside of module files, so we have to do this as a workaround. If you think the current workaround is insufficient, I could separate the AWS storage information into its own module so that the additional resources section is closer to the networking info.

Reply:
> we can't put cross-references inside of module files, so we have to do this as a workaround

oh. Is there a bug report/RFE to allow hrefs everywhere? That's the basis of the WWW.

Reply:
Yup, it's a long-standing issue that frustrates us all. It's related to our repo structure. We hope that the publication platform will eventually support xrefs in modules but I'm not sure about an official RFE. I'll try to find out, because you never know, support from outside CCS might help :)